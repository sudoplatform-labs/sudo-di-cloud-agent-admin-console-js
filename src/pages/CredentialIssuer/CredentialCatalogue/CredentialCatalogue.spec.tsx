import { shallow } from 'enzyme';
import React from 'react';
import { CredentialCatalogue } from '.';

describe('pages/CredentialCatalogue', () => {
  it('Should render', () => {
    const wrapper = shallow(<CredentialCatalogue />);
    expect(wrapper).toMatchSnapshot();
  });
});
