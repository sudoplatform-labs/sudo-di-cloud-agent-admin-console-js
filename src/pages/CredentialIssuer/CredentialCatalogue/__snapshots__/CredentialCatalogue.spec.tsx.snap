// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages/CredentialCatalogue Should render 1`] = `
<styled.div>
  <styled.h1>
    Verifiable Credentials Catalogue
  </styled.h1>
  <SchemaDefinitionsCard />
  <CredentialDefinitionsCard />
</styled.div>
`;
